Apps/Database/src/Migrations/20201118002050_AddResourceDelegate.cs

```diff
@@ -0,0 +1,216 @@
+﻿//-------------------------------------------------------------------------
+// Copyright © 2020 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+#pragma warning disable CS1591
+// <auto-generated />
+using System;
+using System.Text.Json;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace HealthGateway.Database.Migrations
+{
+    public partial class AddResourceDelegate : Migration
+    {
+        private const string Schema = "gateway";
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            // Drop the UserDelegate
+            migrationBuilder.Sql(@$"DROP TABLE IF EXISTS {Schema}.""UserDelegate""");
+
+            // Drop the UserDelegateHistory
+            migrationBuilder.Sql(@$"DROP TABLE IF EXISTS {Schema}.""UserDelegateHistory""");
+
```

Comment:
What about the trigger and the function?

Reply:
I am guessing it gets removed alongside the table.

Reply:
Doubled checked and they are left if not taken manually.  Latest commit removes them